[ACM-11953]: Remove/replace references to hypershift CLI (#6521)
open-cluster-management/rhacm-docs: clusters/hosted_control_planes/install_hcp_cli.adoc

[#hosted-install-cli]
= Installing the hosted control plane command line interface

> The hosted control plane command line interface, `hcp`, is a tool that you can use to get started with hosted control planes. For Day 2 operations, such as management and configuration, use GitOps or your own automation tool.

Reviewer: Optional:

It can be helpful to lead with the "reason" or "why" for the users's action. 

Example: 

```suggestion
To get started with hosted control planes, use the e hosted control plane command line interface, `hcp`, tool. For Day 2 operations, such as management and configuration, use GitOps or your own automation tool.
```
